docs/samples/custom/kfserving-sdk/README.md

```diff
@@ -0,0 +1,73 @@
+# Predict on a InferenceService using the KFServing sdk
```

Comment:
Maybe rename `KFServing sdk` to `KFServing Model Server`? KFServing sdk also could be the client library which may confuse our users.